### PR TITLE
Fix NRE in OverUnderBox

### LINKED ISSUE #275: OverUnderBox ctor may dereference scriptBox parameter if it's null
Noticed while trying to add C# 8 nullability information:

```
            this.Height = baseBox.Height + (over ? delimeterBox.Width : 0) +
                (over && scriptBox != null ? scriptBox.Height + scriptBox.Depth + kern : 0);
            this.Depth = baseBox.Depth + (over ? 0 : delimeterBox.Width) +
                (!over && scriptBox == null ? 0 : scriptBox.Height + scriptBox.Depth + kern);
```

The lower two lines look fishy and the condition may have indended to use `||` instead, but I can't tell for sure.

### PR DESCRIPTION
**Pull request by ForNeVeR**
Closes #275.

### CODE REVIEW

File: src/WpfMath/Boxes/OverUnderBox.cs [hunk 1]
```diff
@@ -3,24 +3,24 @@
...
 
             // Calculate dimensions of box.
             this.Width = baseBox.Width;
-            this.Height = baseBox.Height + (over ? delimeterBox.Width : 0) +
+            this.Height = baseBox.Height + (over ? delimiterBox.Width : 0) +
                 (over && scriptBox != null ? scriptBox.Height + scriptBox.Depth + kern : 0);
-            this.Depth = baseBox.Depth + (over ? 0 : delimeterBox.Width) +
-                (!over && scriptBox == null ? 0 : scriptBox.Height + scriptBox.Depth + kern);
+            this.Depth = baseBox.Depth + (over ? 0 : delimiterBox.Width) +
+                (!over && scriptBox != null ? scriptBox.Height + scriptBox.Depth + kern : 0.0);
```
Comment (gsomix) [inline src/WpfMath/Boxes/OverUnderBox.cs:23]: For consistency.

```suggestion
                (!over && scriptBox != null ? scriptBox.Height + scriptBox.Depth + kern : 0);
```

Comment (ForNeVeR) [reply src/WpfMath/Boxes/OverUnderBox.cs:23]: Usually I prefer the other way around, but that's a good suggestion. I'll change every `0` into `0.0` in this block :3

File: src/WpfMath.Tests/OverUnderBoxTests.fs [hunk 2]
```diff
@@ -32,3 +28,7 @@ type OverUnderBoxTests() =
     member _.``OverUnderBox rendering calls to RenderTransformed for delimiter``() =
         overUnderBox.RenderTo(mockedRenderer, x, y)
         Mock.Verify(<@ mockedRenderer.RenderTransformed(delimiterBox, any(), any(), any()) @>, once)
+
+    [<Fact>]
+    member _.``OverUnderBox should be successfully constructed a box for over == false and scriptBox == null``() =
```
Comment (gsomix) [inline src/WpfMath.Tests/OverUnderBoxTests.fs:33]:
```suggestion
    member _.``OverUnderBox should be successfully constructed for over == false and scriptBox == null``() =
```

Comment (ForNeVeR) [reply src/WpfMath.Tests/OverUnderBoxTests.fs:33]: Also, I'll replace `==` with `=`, since this is F#.

**Comment by ForNeVeR** [comment]
Thanks for the review!

**Pull request merged**